torch/ao/quantization/observer.py

```diff
@@ -1204,7 +1212,8 @@ def reset_histogram(self, x: torch.Tensor, min_val: torch.Tensor, max_val: torch
             x, self.bins, min=min_val, max=max_val, out=self.histogram  # type: ignore[arg-type]
         )
 
-    def forward(self, x_orig: torch.Tensor) -> torch.Tensor:
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
```

Comment:
nit: maybe just keep the old name, L1219 also reassigns x_orig.detach() to x again, seems a bit confusing to assign between x_orig and x back and forth

Reply:
Changing it back then - VScode was complaining the signature doesn't match up with the base class!

Reply:
I just added a pyre-ignore and kept the interface the same. LMK if you prefer the renaming instead.